source/_patterns/01-molecules/tag-list/tag-list.twig

```diff
@@ -18,10 +18,12 @@
   {%- with list only -%}
     {%- block list -%}
 
-      {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['tag-list__list']) }) -%}
+      {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['item-tag']) }) -%}
 
       {%- if singleLine|default(false) -%}
-        {%- set attributes = attributes|merge({class: attributes.class|merge(['tag-list__list--single-line']) }) -%}
+        {%- set attributes = attributes|merge({class: attributes.class|merge(['item-tag__single-line']) }) -%}
+      {%- elseif itemDetails|default(false) -%}
+        {%- set attributes = attributes|merge({class: attributes.class|merge(['item-tag__item-details']) }) -%}
```

Comment:
Still not entirely convinced this is the same pattern (https://github.com/libero/pattern-library/pull/64#issuecomment-470028079), since we need reasonably significant styling differences.

Reply:
Can we make a decision on this whether this is going to be it's own pattern or continue to be in this one.

Reply:
According to @chugginselifesciences , the unlinked "JEL Classification" part may now be handled differently so can be discounted from consideration here. This leaves two categories of thing to consider:

1. the blue items in a list, always comma-delimited, always linked, found in the tag list at the bottom of an article and at the top of teasers and content headers, containing tags, and
1. the grey items in a list, non-delimited, sometimes linked, found at the bottom of content headers and teasers, comprising the article type and a date.

The differences of link state, delimiter and colour between these use cases suggest they are better as two different patterns, rather than one pattern with variants.